src/components/content-tab-assistant.tsx

```diff
@@ -23,25 +24,129 @@ interface MessageProps {
 	className?: string;
 }
 
-export const Message = ( { children, isUser, className }: MessageProps ) => (
-	<div className={ cx( 'flex mt-4', isUser ? 'justify-end' : 'justify-start', className ) }>
-		<div
-			className={ cx(
-				'inline-block p-3 rounded-sm border border-gray-300 lg:max-w-[70%] select-text whitespace-pre-wrap',
-				! isUser && 'bg-white'
-			) }
-		>
-			{ typeof children === 'string' ? (
-				<div id="assistant-markdown">
-					<Markdown>{ children }</Markdown>
-				</div>
-			) : (
-				children
-			) }
+interface InlineCLIProps {
+	output: string;
+	status: 'success' | 'error';
+	time: string;
+}
```

Comment:
This interface is likely to change based on the final shape of the wp-cli implementation in #203 -- just chose some probable params for UI demonstration purposes